docs/examples/kotlin/health/get-queue-functions.md

```diff
@@ -8,4 +8,5 @@ val client = Client(context)
 
 val health = Health(client)
 
-val response = health.getQueueFunctions()
+val response = health.getQueueFunctions(
```

Comment:
Unwanted linebreak was added

Reply:
Not caused by this release; will address in the next (will need to fix for almost all SDKs where a method has **no** required params)